run.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+exec gunicorn --config gunicorn_config.py app.wsgi:todo_app
```

Comment:
You've copied this file into the image but then you seem to ignore it and just use the entrypoint to start the app directly - so can we delete this? And the `gunicorn_config.py`?